oscore-cf/src/main/java/org/eclipse/californium/oscore/OSCoreEndpoint.java

```diff
@@ -0,0 +1,67 @@
+/*******************************************************************************
```

Comment:
will be obsolete using CoapStackFactory. So, please revert this change.

Reply:
Will be removed in re designed PR.